SPECS/kata-containers-cc/mariner-coco-build-uvm-image.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+set -o errexit
+
+[ -n "${DEBUG:-}" ] && set -o xtrace
+
+readonly SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+readonly IMAGE_BUILD_ROOT=`mktemp --directory -t mariner-coco-build-uvm-image-XXXXXX`
+
+generate_image()
+{
+    mv ${ROOT_FOLDER}/nsdax.gpl.c "${IMAGE_BUILD_ROOT}"
+    sudo cp -r ${ROOT_FOLDER}/opt ${IMAGE_BUILD_ROOT}
+    cd "${IMAGE_BUILD_ROOT}"
+
+    # move rootfs
+    sudo mv $ROOT_FOLDER/rootfs-cbl-mariner ${IMAGE_BUILD_ROOT}/rootfs-cbl-mariner
+
+    gcc -O2 ./nsdax.gpl.c -o ${IMAGE_BUILD_ROOT}/nsdax
+
+    # build image
+    sudo DEBUG=1 \
```

Comment:
do we still need DEBUG?

Reply:
This is only for log verbosity and doesn't affect the binary